.gitmodules

```diff
@@ -50,3 +50,6 @@
 [submodule "third-party/small"]
 	path = third-party/small
 	url = https://github.com/transmission/small.git
+[submodule "third-party/rapidjson"]
+	path = third-party/rapidjson
+	url = https://github.com/transmission/rapidjson.git
```

Comment:
Maybe it's worth to add
```
	fetchRecurseSubmodules = false
```
?
This dep fetches another copy of `googletest`.

Reply:
> Another option would be to edit/remove .gitmodules in transmission/rapidjson fork.

I don't have strong opinions on de-duping GTest (it's only 4 MB so :man_shrugging:) but I'd vote against this option on the basis of avoiding unnecessary changes in our forks.

Reply:
6702047

Reply:
I don't think `fetchRecurseSubmodules = false` is working the way we want.

`git clone --recurse-submodules` does not pull RapidJSON, and `git submodule update --init --recursive` pulls the nested gtest anyways.

I will create a PR to remove this config.

Reply:
Nevermind I got confused, it does work.